README.rst

```diff
@@ -75,21 +64,68 @@ Please, cite the software, if you are using for your research.
 Installation
 ============
 
-Conda/Anaconda
---------------
+First install conda : https://docs.conda.io/en/latest/index.html
+
+User
+----
+
+Create a new environment with phenomenal installed in there :
 
 .. code:: shell
 
-    conda install -c openalea openalea.phenomenal
+    conda create -n phm -c conda-forge -c openalea openalea.phenomenal
+    conda activate phm
 
-On windows you need to add the conda-forge channels to find opencv package
+In a existing environment :
 
 .. code:: shell
 
-    conda install -c openalea -c conda-forge openalea.phenomenal
+    conda install -c conda-forge -c openalea openalea.phenomenal
+
+(Optional) Test your installation :
+
+.. code
+
+    conda install -c conda-forge pytest
+    git clone https://github.com/openalea/phenomenal.git
+    cd phenomenal/test; pytest
+
+From source
+-----------
+
+.. code::
+
+    # Install dependency with conda
+    conda create -n phm -c conda-forge python=3
+    conda activate phm
+    conda install -c conda-forge -c openalea cython numpy numba scipy scikit-image scikit-learn networkx=2.3 opencv matplotlib vtk pytest
```

Comment:
Why networkx 2.3? We can add an issue if we want to use the latest version on conda-forge

Reply:
ok

Reply:
#21